news/2017-08-17-happystick-summer-tour-concludes.md

```diff
@@ -0,0 +1,50 @@
+
+![](/wiki/shared/2017-08-17-happystick-summer-tour-concludes/banner.jpg)
+
+As the dust settled on the Winners and Losers Finals of the double-elimination tournament, only one of the 26 Grand Finalists was a repeat from the Spring Tour, leaving us in the thrilling position of witnessing all-new champions across the board. And thrilled we were, as seven of the 13 matches would go to a bracket reset as the players fighting their way from the Losers Finals sought to show that they deserved to be there just as much as those who hadn’t dropped a match and advanced from the Winners Brackets. Would the favorites find victory, or would the underdogs claw their way through the grueling task of winning back to back best of 11-map series to find glory? Read on for results and VODs of our featured matches!
+
+<p align="center"><iframe src="https://player.twitch.tv/?video=v167002207&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="620"></iframe></p>
+
+The [250 - 500 rank bracket](http://challonge.com/znsjvw11) saw well-known all-around player [-GN](https://osu.ppy.sh/u/895581) pitted against [smh](https://osu.ppy.sh/u/1629553), a former tourney winner himself in the 2016 osu! Indonesia Tournament. In a surprising result in the Winners Final, smh had knocked -GN down to the Losers Bracket in a back-and-forth clash where he would ultimately triumph in a tiebreaker, and as such -GN entered the Grand Final seeking revenge. The match would prove to be a strategic one as early as the ban stage: recognizing smh to be a skilled Double Time player, -GN banned Approach Rate 10.3 DT pick [Utagumi Setsugekka - Yumemi Sunrise](https://osu.ppy.sh/b/338862), while smh in turn targeted -GN’s unparalleled reading ability by banning [Eisyo-kobu - Oriental Blossom](https://osu.ppy.sh/b/1242790). As the winner of the roll, -GN selected the first map, but despite having his choice he was defeated on [xi - Aragami](https://osu.ppy.sh/b/547874). The players would split the next two picks before smh managed to hold his own on technically challenging [Camellia vs Akira Complex - Reality Distortion](https://osu.ppy.sh/b/1171267) to take the lead at 3-1. Both players won their next two picks each, leaving it match point for smh at 5-3. -GN selected Hidden map [ginkiha - Anemoi](https://osu.ppy.sh/b/1316661), and it would be here that smh seized his opportunity as -GN broke partway through the map while smh held his combo to an insurmountable lead to seal a 6-3 victory.
```

Comment:
new paragraph at "the match would prove"

new paragraph at the players would split the next two picks

some minor wording stuff could be changed to shorten this up but it's alright overall